docs/recipes/navbar.md

```diff
@@ -16,22 +16,136 @@ In this example we want the navbar to stick to the top of the page, so we will u
  prop to make the navbar fixed to the top of the page.
 We will also use the `top= 0` and `z_index="1"` props to make sure the navbar is always on top of the screen and above the other components on the page.
```

Comment:
say `z_index="1"` in the text but then have `z_index="5"` in the code. Also maybe quickly explain what the z_index and how it makes sure that it is always at the top